drizzlepac/haputils/hapcut_utils.py

```diff
@@ -0,0 +1,383 @@
+from stsci.tools import logutil
+
+import math
+import numpy as np
+import os
+import shutil
+import sys
+
+__taskname__ = 'hapcut_utils'
+
+MSG_DATEFMT = '%Y%j%H%M%S'
+SPLUNK_MSG_FORMAT = '%(asctime)s %(levelname)s src=%(name)s- %(message)s'
+log = logutil.create_logger(__name__, level=logutil.logging.NOTSET, stream=sys.stdout, 
+                            format=SPLUNK_MSG_FORMAT, datefmt=MSG_DATEFMT)
+
+
+def mvm_id_filenames(sky_coord, cutout_size, verbose=False):
+    """
+    This function retrieves a table of MVM drizzled image filenames with additional
+    information from the archive.  The user can then further cull the table to use as
+    input to obtain a list of files from the archive.
+
+    Parameters
+    ----------
+    sky_coord : str or `~astropy.coordinates.SkyCoord` object
+        The position around which to cutout. It may be specified as a string ("ra dec" in degrees)
+        or as the appropriate `~astropy.coordinates.SkyCoord` object.
+
+    cutout_size : int, array-like, `~astropy.units.Quantity`
+        The size of the cutout array. If ``cutout_size`` is a scalar number or a scalar
+        `~astropy.units.Quantity`, then a square cutout of ``cutout_size`` will be created.
+        If ``cutout_size`` has two elements, they should be in ``(ny, nx)`` order.  Scalar numbers
+        in ``cutout_size`` are assumed to be in units of arcseconds. `~astropy.units.Quantity` objects
+        must be in angular units.
+
+    verbose : bool
+        Default False.  If True, additional intermediate information is printed.
+
+    Returns
+    -------
+    filtered_dp_table : `~astropy.table.Table` object
+
+    """
+
+    # If the cutout_size is not an astropy.units.Quantity object, the scalar(s)
+    # are assumed to be arcseconds.  The variable must be cast as a Quantity.
+    if not isinstance(cutout_size, Quantity):
+        cutout_size *= u.arcsec
+        cutout_size = np.atleast_1d(cutout_size)
+        if len(cutout_size) == 1:
+            cutout_size = np.repeat(cutout_size, 2)
+
+    if not isinstance(sky_coord, SkyCoord):
+        sky_coord = SkyCoord(sky_coord, unit="deg")
+
+    # From HST data, Search for the list of images based upon: coordinates, search region, data
+    # product type, and the instrument name (with wildcard), project (HAP), and observation
+    # collection (HST).  Use the wildcard to get all the detectors for the instrument.  Multiple
+    # instruments cannot be searched at the same time.  Use the diagonal of the cutout to define
+    # the search radius for the archive.  Images which fall outside the desired cutout need to
+    # be filtered from the solution later.
+    radius = math.ceil(math.sqrt(math.pow(cutout_size.value[0], 2) + math.pow(cutout_size.value[1], 2)) / 2.0)
+
+    # Careful - the radius must be a str or Quantity
+    radius *= u.arcsec
+    if verbose:
+        print("Performing query for ACS images. Radius: {}.".format(radius))
```

Comment:
The logging has been defined at the top of the module.  Subsequent print statements should at least be accompanied by the relevant 'log.info' or 'log.warning' or 'log.debug' statements.  Alternatively, the print statements could be replaced entirely by the log statements with the log level set based on the value of verbose as done in other modules.

Reply:
Use logging as is done for the HAP tasks.  The "verbose" parameter only applies to the underlying "astrocut" functionality.